### PR TITLE
Create the file structure

### PR DESCRIPTION
**Pull request by yongmin86k**
# Create the file structure

- Create screen folders
- Rename the app as `BC Parks Foundations`

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  whitespace = red reverse
+[color]
+    ui = auto
+[alias]
+    st = status
+    ci = commit --verbose
+    co = checkout
+    di = diff -w
+    dc = diff --cached -w
+    noff = merge --no-ff
```
Comment (siddharthparmar7) [inline .gitconfig]: This file should be stored in your $HOME dir. Please check https://github.com/redacademy/bcf-fall-2019/wiki/Git-Merge-Workflow

File: .gitignore [hunk 2]
```diff
@@ -56,4 +60,10 @@ buck-out/
...
 
 # CocoaPods
-/ios/Pods/
+**/ios/Pods/
+*Podfile.lock
+
+**/dist
+**/.vscode
+**/.log
+**/.env
```
Comment (siddharthparmar7) [inline .gitignore]: This file should not be here as it was part of the other pull request.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# BC Parks Foundation
+
+## Authors
+
+- Alex
+- Matt
+- Yongmin
```
Comment (siddharthparmar7) [inline README.md]: same as above.

File: client/js/layouts/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+// Add any code here
```
Comment (siddharthparmar7) [inline client/js/layouts/index.js]: What is this dir for?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
yarn.lock should not be ignored. Please make the changes. Thank you.

File: .gitconfig (hunk 1 above)
Comment (yongmin86k) [reply .gitconfig]: oki

File: client/js/layouts/index.js (hunk 4 above)
Comment (yongmin86k) [reply client/js/layouts/index.js]: For the same structure screens but different data fetching from API

File: .gitignore (hunk 2 above)
Comment (yongmin86k) [reply .gitignore]: I made this commit after I worked on the `.gitignore` file.

File: README.md (hunk 3 above)
Comment (yongmin86k) [reply README.md]: Yes

File: client/js/layouts/index.js (hunk 4 above)
Comment (siddharthparmar7) [reply client/js/layouts/index.js]: I don't think you need this dir. We can add it later if needed. API calls should happen in the container file.

File: .gitconfig (hunk 1 above)
Comment (siddharthparmar7) [reply .gitconfig]: Please remove this file from here and add it to your $HOME.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Thanks, Yongmin. Please make the requested changes.

File: .gitconfig (hunk 1 above)
Comment (yongmin86k) [reply .gitconfig]: Done😄

File: client/js/layouts/index.js (hunk 4 above)
Comment (yongmin86k) [reply client/js/layouts/index.js]: Ok, I removed it

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts. Thank you.

File: .gitconfig (hunk 1 above)
Comment (siddharthparmar7) [reply .gitconfig]: Thank you for deleting this file in your other branch. You should delete it from here too.

**Review by siddharthparmar7 — APPROVED** [review]
Nice work!

**Pull request merged**